### PR TITLE
Hw4_Oleksii_Fedorchuk

### PR DESCRIPTION
**Pull request by Oleksii-Fedorchuk**

### CODE REVIEW

File: Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task1.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+list_of_elements = [1, 2, 3, 4, 5, 6, 7, 8]
+odd_numbers = []
+even_numbers = []
+for index, number in enumerate(list_of_elements):
+    if index % 2 == 0:
+        odd_numbers.append((index, number))
+    else:
+        even_numbers.append((index, number))
+print(f"Odd numbers are: {odd_numbers}")
+print(f"Even numbers are: {even_numbers}")
```
Comment (bohdandovbsyh) [inline Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task1.py:10]: empty line

File: Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task2.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+a *= b
+print(a)
+
+# Dividing
+
+result2 = a / b
+print(result2)
+
+a /= b
+print(a)
```
Comment (bohdandovbsyh) [inline Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task2.py:12]: use byte operations for this task

Comment (BohdanDovbysh1994) [reply Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task2.py:18]: empty line

File: Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task3.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Jonathan", "Artur"]
+
+for friend in friends:
+    if friend in enemies:
+        print(f"{friend}  we are not the friends anymore")
+    elif friends == "James":
+        continue
+    else:
+        print(f"{friend} we are the best friends")
```
Comment (bohdandovbsyh) [inline Oleksii_Fedorchuk/HW4_Oleksii_Fedorchuk/task3.py:10]: empty line

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**